### ISSUE TITLE
A11y_VSCode_Version Control_Screenreader: Screenreader does not read the error message while creating new file or folder with empty name

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/vscode#99422]
[Check out Accessibility Insights!](https://github.com/microsoft/AzureStorageExplorer/issues/url) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_VSCode;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#DesktopApp;#MAS1.3.1;#MAS3.3.1;#Win10;#Visual Studio Code;#NVDA;#JAWS; #CAITestPass_May2020;#FTP;#STP;

**Environment Details:**
Application Name: Visual Studio Code
Windows Version: Windows10
Screenreader version:NVDA[2020.1], JAWS 2020.2001.70.400
**Repro Steps:**
1.Open Visual studio code editor
2.Using F1 key navigate to the palette, type "git branch" and select "Create Branch"
3. Enter the branch name in the edit field and hit enter. Observe the change in branch name at the bottom left corner
4. Navigate to open folder and click on New file button.
5.Open screenreader and check whether it reads the error message as "A file or folder name must be provided.
**Actual:**
While navigating through keyboard,Screenreader does not read the error message as "A file or folder name must be provided" while creating new file or folder with empty name.

It is only reading for the first time .
This issue is reproduced with NVDA and Jaws.

**Expected:**
Screenreader should read the error message as "A file or folder name must be provided"while creating new file or folder with empty name.

**Attachment link for Reference:**
![NVDA](https://user-images.githubusercontent.com/53940787/83841518-9e9a6f00-a71e-11ea-8f8f-179d5a3b6538.gif)

https://microsoft-my.sharepoint.com/:v:/p/v-nisaho/EVHF_7m6NC5MhlIlK9TLRfkBp291YzQ84EzxYffOQ-kCGw?e=AcFFVI
